Resources/views/FragmentAdmin/form.html.twig

```diff
@@ -9,7 +9,7 @@
                     </h4>
                 </div>
                 <div class="box-body">
-                    {% if form.vars.errors|length > 0 %}
+                    {% if form.vars.errors is defined and form.vars.errors|length > 0 %}
```

Comment:
`form.vars.errors|default([])|lenght` would be shorter, but the best would be to have this variable always defined